src/omnicore/sp.h

```diff
@@ -121,6 +122,7 @@ class CMPSPInfo : public CDBBase
             READWRITE(update_block);
             READWRITE(fixed);
             READWRITE(manual);
+            READWRITE(unique);
```

Comment:
This requires a reparse and thus bumping the DB version.

edit: nevermind, saw you bumped it later. :)